Makefile.one

```diff
@@ -8,11 +8,10 @@ all:
 
 PARALLEL = $(shell $(CMD_GREP) -c ^processor /proc/cpuinfo)
 
-NORM_MAKE = make -j$(PARALLEL)
+NORM_MAKE = make
 
 MAKE = make -f Makefile.one
 MAKEFLAGS += --no-print-directory
-MAKEFLAGS += -j$(PARALLEL)
```

Comment:
This was causing issues such as:

```
make[1]: warning: -j24 forced in makefile: resetting jobserver mode.
```

So its better to use $(PARALLEL) whenever we would like builds to be parallel by default (or to rely in user providing -jX to make as an argument).